iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
thanks I was trying to run the tests locally with wct - that thing doesn't want to work with my setup, but you've beat me to it figuring out the issue. I'll make this fix and commit once more